cadc-inventory/src/main/java/org/opencadc/inventory/InventoryUtil.java

```diff
@@ -89,9 +95,36 @@
 public abstract class InventoryUtil {
     private static final Logger log = Logger.getLogger(InventoryUtil.class);
 
+    public static final String CERTIFICATE_FILE_LOCATION = System.getProperty("user.home") + "/.ssl/cadcproxy.pem";
+
```

Comment:
I can foresee use cases where some components have configured cert files and cannot operate with a single file in default location, so I'd prefer to leave this default in the individual components and not entrench this idea in the libraries with by defining constants and adding convenience methods.

So, components (critwall and fenwick currently) should call a `renewSubject(Subject, File)` method directly.